app/Service/StudentDetailService.php

```diff
@@ -21,15 +21,16 @@ public function execute($student)
     public function getStudentDetailsById($studentId){
         
         $resume = Resume::where('student_id', $studentId)->first();
```

Comment:
```suggestion
        Resume::where('student_id', $studentId)->first();
```

Reply:
Done. Thank you very much for the suggestions.